### PR TITLE
Conditional route handling  

### PR DESCRIPTION
**Pull request by Aradhya-Tripathi**
Route handling and 404 rendering

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jagnani73 — APPROVED** [review]

File: server/githubsrm/core/settings.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 SECRET_KEY = os.getenv('SECRET_KEY')
 
-DEBUG = False
+DEBUG = True
```
Comment (gitaalekhyapaul) [inline server/githubsrm/core/settings.py:25]: `DEBUG` should be `False`

**Review by gitaalekhyapaul — CHANGES_REQUESTED** [review]
Please incorporate the changes

**Comment by Aradhya-Tripathi** [comment]
> Please incorporate the changes

Resolved in 57a8072

**Review by jagnani73 — APPROVED** [review]

**Review by gitaalekhyapaul — APPROVED** [review]
LGTM :+1:

**Pull request merged**